runtime/interpreter/interpreter_expression.go

```diff
@@ -169,6 +172,12 @@ func (interpreter *Interpreter) memberExpressionGetterSetter(memberExpression *a
 
 	isNestedResourceMove := interpreter.Program.Elaboration.IsNestedResourceMoveExpression(memberExpression)
 
+	memberInfo, ok := interpreter.Program.Elaboration.MemberExpressionMemberInfo(memberExpression)
```

Comment:
Just related: Maybe we should improve the naming of `MemberInfo` to e.g. `MemberAccessInfo` or just `MemberAccess`, as it is the info for a particular access, not the member in general. 

I was confused for a moment why `ReturnReference` was added there, thinking it would apply for any member usage (but it isn't)

Reply:
Renamed in 84e7e0fd759fed08fdbd04e0b0be0b8d722b8eed